ingestion_tools/scripts/migrate_folder_structure.py

```diff
@@ -0,0 +1,396 @@
+    "tiltseries_metadata": "{dataset_name}/{run_name}/TiltSeries/tiltseries_metadata.json",
+    "tomogram": "{dataset_name}/{run_name}/Tomograms/VoxelSpacing{voxel_spacing_name}/CanonicalTomogram",
+    "tomogram_metadata": "{dataset_name}/{run_name}/Tomograms/VoxelSpacing{voxel_spacing_name}/CanonicalTomogram/"
+                         "tomogram_metadata.json",
+    "viz_config": "{dataset_name}/{run_name}/Tomograms/VoxelSpacing{voxel_spacing_name}/CanonicalTomogram/"
+                  "neuroglancer_config.json",
+    "voxel_spacing": "{dataset_name}/{run_name}/Tomograms/VoxelSpacing{voxel_spacing_name}",
+}
+
+
+def move(config: DepositionImportConfig, old_path: str, new_path: str):
+    if config.fs.exists(old_path):
+        print(f"Moving {old_path} to {new_path}")
+        config.fs.move(old_path, new_path, recursive=True)
+    else:
+        print(f"Skipping move of {old_path} as it does not exist")
+
+
+def delete(config: DepositionImportConfig, path: str):
+    print(f"Deleting {path}")
+    if hasattr(config.fs, "s3fs"):
+        config.fs.s3fs.rm(path)
+
+
+def migrate_volume(
+        cls, config: DepositionImportConfig, parents: dict[str, Any], kwargs,
+) -> None:
+    """
+    Moves the volumes of the entity to include the identifier in the path (default of 100)
+    Moves the metadata of the entity  to include the identifier in the path (default of 100)
+    :param cls:
+    :param config:
+    :param parents:
+    :param kwargs:
+    :param migrate_metadata:
+    :return:
+    """
+    output_path = cls.get_output_path()
+    metadata_path = kwargs.get("metadata_path")
+    metadata_dir = os.path.dirname(metadata_path)
+    for fmt, key in {"mrc": "mrc_files", "zarr": "omezarr_dir"}.items():
+        filename = cls.metadata.get(key)
+        if isinstance(filename, list):
+            filename = filename[0]
+        old_path = os.path.join(metadata_dir, filename)
+        new_path = f"{output_path}.{fmt}"
+        move(config, old_path, new_path)
+    if kwargs.get("migrate_metadata", True):
+        move(config, metadata_path, cls.get_metadata_path())
+
+
+def migrate_tomograms(cls, config: DepositionImportConfig, parents: dict[str, Any], kwargs) -> None:
+    """
+    Moves the tomograms in the run to include the identifier in the path (default of 100)
+    Moves the tomogram_metadata in the run to include the identifier in the path (default of 100)
+    Update the path to Reconstruction/VoxelSpacingXX.XX/Tomograms/...
+
+    Update the tomogram path in the metadata json
+    Update the key_photos file path in the metadata json
+    Update the neuroglancer_config path name in the metadata json
+    :param cls:
+    :param config:
+    :param parents:
+    :param kwargs:
+    :return:
+    """
+    migrate_volume(cls, config, parents, {**kwargs, "migrate_metadata": False})
+    # This is a very hacky way to get the metadata path set up with the correct values
+    cls.allow_imports = True
+    cls.import_metadata()
```

Comment:
Note - Please doublecheck that `key_photo` paths in `tomogram_metadata.json` are getting updated.